### PR TITLE
[`core`] Fix quality issue

### PR DESCRIPTION
**Pull request by younesbelkada**
Fixes quality check that were not being run on the examples to avoid extra files being modified such as in https://github.com/lvwerra/trl/pull/190

cc @lvwerra

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -4,7 +4,7 @@ test:
 	python -m pytest -n auto --dist=loadfile -s -v ./tests/
 
 quality:
-	black --check --line-length 119 --target-version py38 tests trl
+	black --check --line-length 119 --target-version py38 tests trl examples
 	isort --check-only tests trl
 	flake8  tests trl
```
Comment (lvwerra) [inline Makefile:10]: Let's apply all tests to the same files:

```suggestion
	black --check --line-length 119 --target-version py38 tests trl examples
	isort --check-only tests trl examples
	flake8  tests trl examples

```

Comment (lvwerra) [reply Makefile:10]: Let's do the same in `style` and remove `setup.py`

**Review by lvwerra — COMMENTED** [review]
Small nit, otherwise good.

File: Makefile (hunk 1 above)
Comment (younesbelkada) [reply Makefile]: Proposed something in [4ad0581](https://github.com/lvwerra/trl/pull/197/commits/4ad05816af684111d5a867e43db086fffb2b37f8)

Comment (lvwerra) [reply Makefile]: much cleaner indeed :)

**Review by lvwerra — APPROVED** [review]

**Pull request merged**